src/plugins/uk.ac.kcl.mdeoptimise.ui/plugin.xml

```diff
@@ -434,12 +434,12 @@
 		</launchConfigurationTabGroup>
 	</extension>
 	<extension point="org.eclipse.jdt.core.classpathContainerInitializer">
-		<classpathContainerInitializer id="MDEOptimiser" class="uk.ac.kcl.ui.classpath.MDEOClassPathContainerInitializer" />
+		<classpathContainerInitializer id="uk.ac.kcl.mde_optimise.MDEO_CONTAINER" class="uk.ac.kcl.ui.classpath.MDEOClassPathContainerInitializer" />
 	</extension>
 	<extension point="org.eclipse.jdt.ui.classpathContainerPage">
 		<classpathContainerPage
 			id="uk.ac.kcl.mdeoptimise.ui"
-			name="MDEOptimiser System Library"
+			name="MDEOptimiser Core Library"
```

Comment:
I see you have systematically renamed this to "Core" library. I'm not fully sure I understand the difference between the two, but thought that "System" was the correct terminology for what we provide?

In any case, you probably then also need to adjust the type constant returned by the actual container provider, don't you?

Reply:
I thought that Core is better than System in this case because System made me think of a dependency between the OS and MDEO. I feel that Core is clearer, referring to core functionality.

Reply:
I see. But Eclipse JavaCore seems to explicitly distinguish Core and System libraries. We should make sure we know what each of them is and use the correct terminology.